README.md

```diff
@@ -72,7 +71,7 @@ These are all the configuration settings (can also be viewed with the `--help` s
 							virt-workshop
 	  --master-flavor <ssc.small>
 							The openstack flavor for the master node, default is
-							ssc.small
+							ssc.small.
```

Comment:
No period after the help text. No other help text has this from the command lines.

Reply:
What a picky reviewer!